tfjs-backend-webgpu/src/backend_webgpu_test.ts

```diff
@@ -246,3 +246,128 @@ describeWebGPU('backendWebGPU', () => {
             2);  // One is the storage buffer, the other is the staging buffer.
   });
 });
+
+describeWebGPU('keeping data on gpu ', () => {
+  let flag: boolean;
+
+  beforeAll(() => {
+    flag = tf.env().getBool('WEBGPU_CPU_FORWARD');
+    tf.env().set('WEBGPU_CPU_FORWARD', false);
+  });
+
+  afterAll(() => {
+    tf.env().set('WEBGPU_CPU_FORWARD', flag);
+  });
+
+  it('has a valid buffer for dtype=float32.', async () => {
+    const webGPUBackend = (tf.backend() as WebGPUBackend);
+    const data = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+    const size = 48;
+    const a = tf.tensor(data, [1, 3, 4]);
+    const b = tf.add(a, 0);
+    const res = b.dataToGPU();
+    expectArraysEqual(res.bufSize, size);
+    const resData = await webGPUBackend.downloadGPUBufferData(res);
+    expectArraysClose(resData, new Float32Array(data));
```

Comment:
Why not directly use `expectArraysClose(resData, data);`?

Reply:
Done